css/properties/animation-timing-function.json

```diff
@@ -0,0 +1,152 @@
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "edge_mobile": [
+              {
+                "version_added": true
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "16"
+              },
+              {
+                "prefix": "-moz-",
+                "version_added": "5"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "49"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "44",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.prefixes.webkit",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "16"
+              },
+              {
+                "prefix": "-moz-",
+                "version_added": "5"
+              },
+              {
+                "version_added": "49",
+                "prefix": "-webkit-"
+              },
+              {
+                "version_added": "44",
+                "prefix": "-webkit-",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.prefixes.webkit",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "opera": [
+              {
+                "version_added": "12.10"
+              },
+              {
+                "prefix": "-o-",
+                "version_added": "12"
+              }
+            ],
+            "opera_android": {
+              "prefix": "-o-",
+              "version_added": "12"
+            },
+            "safari": {
+              "prefix": "-webkit-",
+              "version_added": "4"
+            },
+            "safari_ios": {
+              "prefix": "-webkit-",
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "frames": {
+          "__compat": {
+            "description": "frames()",
+            "support": {
+              "chrome": {
+                "version_added": false,
+                "notes": "The name of the <code>frames()</code> timing function is <a href='https://github.com/w3c/csswg-drafts/issues/1301'>under discussion</a>, so it is disabled in browser release versions until a final decision is reached. It is enabled in Nightly/Canary only."
+              },
+              "chrome_android": {
+                "version_added": true
```

Comment:
This is susipicious. Any source for it?

Reply:
Yes, It is incorrect: false
(See: https://www.chromestatus.com/feature/5189363944128512 )